instrumentation/rocketmq-client-4.8/testing/src/main/groovy/io/opentelemetery/instrumentation/rocketmq/AbstractRocketMqClientLibraryTest.groovy

```diff
@@ -0,0 +1,161 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetery.instrumentation.rocketmq
+
+import base.BaseConf
+import io.opentelemetry.instrumentation.test.InstrumentationSpecification
+import io.opentelemetry.semconv.trace.attributes.SemanticAttributes
+import org.apache.rocketmq.common.message.Message
+import org.apache.rocketmq.remoting.common.RemotingHelper
+import org.apache.rocketmq.test.client.rmq.RMQNormalConsumer
+import org.apache.rocketmq.test.client.rmq.RMQNormalProducer
+import org.apache.rocketmq.test.listener.rmq.concurrent.RMQNormalListener
+import org.apache.rocketmq.test.listener.rmq.order.RMQOrderListener
+import spock.lang.Shared
+import spock.lang.Unroll
+
+import static io.opentelemetry.api.trace.SpanKind.CONSUMER
+import static io.opentelemetry.api.trace.SpanKind.PRODUCER
+import static io.opentelemetry.instrumentation.test.utils.TraceUtils.basicSpan
+import static io.opentelemetry.instrumentation.test.utils.TraceUtils.runUnderTrace
+
+@Unroll
+abstract class AbstractRocketMqClientLibraryTest extends InstrumentationSpecification{
```

Comment:
There's no need to keep that in the common `testing` module if it only tests library instrumentation code.